aws/resource_aws_spot_fleet_request.go

```diff
@@ -377,6 +382,21 @@ func buildSpotFleetLaunchSpecification(d map[string]interface{}, meta interface{
 		}
 	}
 
+	if v, ok := d["tags"]; ok {
```

Comment:
Do you mind adding an extra check here? i.e.

```go
if m, ok := d["tags"].(map[string]interface{}); ok && len(m) > 0 {
```
otherwise this would break all configs without tags, see output from relevant acceptance tests:

```
=== RUN   TestAccAWSSpotFleetRequest_associatePublicIpAddress
--- FAIL: TestAccAWSSpotFleetRequest_associatePublicIpAddress (682.03s)
	testing.go:434: Step 0 error: Error applying: 1 error(s) occurred:

		* aws_spot_fleet_request.foo: 1 error(s) occurred:

		* aws_spot_fleet_request.foo: Error requesting spot fleet: [WARN] Error creating Spot fleet request, retrying: InvalidSpotFleetRequestConfig: Tag value cannot be null. Use empty string instead.
			status code: 400, request id: 42b4061d-3214-4bae-9fba-0751e13458ab
```